src/index.ts

```diff
@@ -4,8 +4,9 @@ import Job from "./automations/Job";
 import SSO from "./automations/SSO";
 import { MAIN_URL, PROTECTED_JOB_BOARDS } from "./common/constants";
 import { joinURL } from "./common/pageUtils";
+import { ADD_POSTS_DESCRIPTION, ADD_POSTS_REGIONS_DESCRIPTION, ADD_POSTS_USAGE, CLONE_FROM_DESCRIPTION, DELETE_POSTS_DESCRIPTION, DELETE_POSTS_REGIONS_DESCRIPTION, DELETE_POSTS_USAGE, GENERAL_DESCRIPTION, INTERACTIVE_DESCRIPTION, JOB_ID_DESCRIPTION, SIMILAR_DESCRIPTION } from "./common/commandDescriptios";
```

Comment:
One little typo:

```suggestion
import { ADD_POSTS_DESCRIPTION, ADD_POSTS_REGIONS_DESCRIPTION, ADD_POSTS_USAGE, CLONE_FROM_DESCRIPTION, DELETE_POSTS_DESCRIPTION, DELETE_POSTS_REGIONS_DESCRIPTION, DELETE_POSTS_USAGE, GENERAL_DESCRIPTION, INTERACTIVE_DESCRIPTION, JOB_ID_DESCRIPTION, SIMILAR_DESCRIPTION } from "./common/commandDescriptions";
```

Alternatively you can do:
```
import * as commands from ...
```
As you prefer